README.md

```diff
@@ -47,28 +47,37 @@ programming and debugging. These are the main features:
 See [the libdragon CLI](https://github.com/anacierdem/libdragon-docker) to
 quickly get libdragon up and running. Basically:
 
-1. Download the CLI (as a pre-built binary, or build from source)
-2. Run `libdragon init` to create a skeleton project
-3. Run `libdragon make` to compile a build a ROM
+1. Make sure that you have Docker installed correctly (on Windows and Mac, use
+   Docker Desktop). You can run `docker system info` to check that it is working
+   correctly.
+2. Install the [the libdragon CLI](https://github.com/anacierdem/libdragon-docker).
+   You have three options:
+
+   1. Download the [pre-built binary](https://github.com/anacierdem/libdragon-docker/releases/tag/v10.8.0), 
+      and copy it into some directory which is part of your system PATH.
+   2. If you have `npm` installed (at least verstion 14), run `npm install -g libdragon`.
+   3. Build it from source (by running `npm install` after cloning the repository).
```

Comment:
This might not be ideal to have here, rather it might be better to just provide a link to the original repository or remove this option to reduce confusion.

Reply:
OK do you prefer to remove the 3 or leave it as "build from the source" but just sends to the repo instead of hinting the command?

Reply:
Done